scribble-lib/help/search.rkt

```diff
@@ -17,38 +17,40 @@
                         #:notify [notify void])
   (define open-url (get-doc-open-url))
   (cond
-   [open-url
-    (define dest-url (let ([u (string->url open-url)])
-                       (combine-url/relative
-                        u
-                        (string-join
-                         (for/list ([s (explode-path sub)])
-                           (if (path? s)
-                               (path-element->string s)
-                               (format "~a" s)))
-                         "/"))))
-    (notify (url->string dest-url))
-    (send-url (url->string
-               (struct-copy url dest-url
-                            [fragment (or fragment
-                                          (url-fragment dest-url))]
-                            [query (append
-                                    (url-query dest-url)
-                                    (if query
-                                        (url-query
-                                         (string->url
-                                          (format "q?~a" query)))
-                                        null))])))]
+    [open-url
+     (define dest-url (let ([u (string->url open-url)])
+                        (combine-url/relative
+                         u
+                         (string-join
+                          (for/list ([s (explode-path sub)])
+                            (if (path? s)
+                                (path-element->string s)
+                                (format "~a" s)))
+                          "/"))))
+     (notify (url->string dest-url))
+     (send-url (url->string
+                (struct-copy url dest-url
+                             [fragment (or fragment
+                                           (url-fragment dest-url))]
+                             [query (append
+                                     (url-query dest-url)
+                                     (if query
+                                         (url-query
+                                          (string->url
+                                           (format "q?~a" query)))
+                                         null))])))]
     [else
-      (let* ([path (build-path (find-user-doc-dir) sub)]
```

Comment:
This one should not be converted (as discussed in Slack).

Reply:
Done. Let me know if you see any other instances of this.